client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -293,6 +293,12 @@ const addDomainAction = ( siteSlug, isStaticHomePage ) => ( dispatch ) => {
 	page( `/domains/add/${ siteSlug }` );
 };
 
+// WPCOM-specific domains like free and staging sub-domains are filtered by `canCurrentUserAddEmail`
```

Comment:
I think comments describing methods/functions are nicer when using JSDOC notation. I'm completely fine with just comment, but we could start adding JSDOC to every new method/func/class

Reply:
Thanks for the catch!